appcues/src/main/java/com/appcues/data/local/room/AppcuesDatabase.kt

```diff
@@ -1,11 +1,18 @@
 package com.appcues.data.local.room
 
+import androidx.room.AutoMigration
 import androidx.room.Database
 import androidx.room.RoomDatabase
 import androidx.room.TypeConverters
 import com.appcues.data.local.model.ActivityStorage
 
-@Database(entities = [ActivityStorage::class], version = 1, exportSchema = false)
+@Database(
+    entities = [ActivityStorage::class],
+    version = 2,
+    autoMigrations = [
+        AutoMigration(from = 1, to = 2)
+    ]
+)
```

Comment:
thus the new `version` here and the `autoMigration` to handle it (newer Room feature as documented [here](https://developer.android.com/training/data-storage/room/migrating-db-versions))

removing `exportSchema = false` is required, which generates the new files in the repo seen here in 
```
appcues/schemas/com.appcues.data.local.room.AppcuesDatabase/1.json
appcues/schemas/com.appcues.data.local.room.AppcuesDatabase/2.json
```
for db versions 1 and 2, and providing the ability to do the diff and migrate this new column.